classes/genre.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :id, :name, :items
```

Comment:
According to the diagram, `id` and `items` must be private. You should use `attr_reader` for them!

Reply:
✅ Done and corrected.